hs_core/templates/pages/create-resource.html

```diff
@@ -55,7 +55,7 @@
                 <option value="SWATModelInstanceResource">SWAT Model Instance</option>
                 <option value="NetcdfResource">Multidimensional (NetCDF)</option>
                 <option value="TimeSeriesResource">Time Series</option>
-{#                <option value="ToolResource">Application</option>#}
+                <option value="ToolResource">Web App</option>

```

Comment:
What about "HydroShare App"?  Web app is a too generic and may be confusing to some users (HydroShare is after all a web app itself).